MediaGalleryUi/Model/GetDetailsByAssetId.php

```diff
@@ -85,6 +85,7 @@ public function execute(int $assetId): array
             'image_url' => $this->getUrl($asset->getPath()),
             'title' => $asset->getTitle(),
             'path' => $asset->getPath(),
+            'description' => $asset->getDescription(),
```

Comment:
Looks like this PR contains parts of https://github.com/magento/adobe-stock-integration/pull/1512 PR. Can you please ensure each PR contains only the changes related to the task